src/SDKs/GuestConfiguration/AzSdk.RP.props

```diff
@@ -3,6 +3,5 @@
   <PropertyGroup>
     <AzureApiTag>Compute_2018-06-30-preview;GuestConfiguration_2018-06-30-preview;</AzureApiTag>
```

Comment:
Was `Compute_2018-06-30-preview` autogenerated?

Reply:
yes, it is autogenerated